src/FHE.sol

```diff
@@ -3,11 +3,9 @@ pragma solidity ^0.8.19;
 
 import "./Bytes.sol";
 
-contract FHE {
```

Comment:
Did you mean to make this change? This has downstream consequences, and there are a number of PRs currently open that address them. I'd probably revert this change, unless we want to merge all those PRs first.

Reply:
Oh whoops no I did not.

Reply:
Fixed.